packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -134,25 +116,77 @@ <h2>
-                    placeholder="{{:: 'pci_projects_project_training_jobs_submit_attach_source_placeholder' | translate }}"
-                ></oui-dual-list-source>
-                <oui-dual-list-target
-                    searchable="true"
-                    heading="{{:: 'pci_projects_project_training_jobs_submit_attach_target_heading' | translate }}"
-                    placeholder="{{:: 'pci_projects_project_training_jobs_submit_attach_target_placeholder' | translate }}"
-                ></oui-dual-list-target>
-            </oui-dual-list>
+                <oui-inline-adder-row>
+                    <oui-inline-adder-field>
+                        <oui-field
+                            data-label="{{:: 'pci_projects_project_training_jobs_submit_data_container' | translate  }}"
+                        >
+                            <label class="oui-select">
+                                <oui-select
+                                    id="container"
+                                    name="container"
+                                    data-model="container"
+                                    data-items="$ctrl.filteredContainers"
+                                    data-match="description"
+                                    data-searchable
+                                    data-required
+                                >
+                                </oui-select>
+                            </label>
+                        </oui-field>
+                    </oui-inline-adder-field>
+                    <oui-inline-adder-field>
+                        <oui-field
+                            data-label="{{:: 'pci_projects_project_training_jobs_submit_data_mount_path' | translate  }}"
+                        >
+                            <input
+                                type="text"
+                                class="oui-input"
+                                placeholder="/workspace/demo"
+                                data-ng-init="mountPath = '/workspace/container_' + $ctrl.job.volumes.length"
+                                name="mountPath"
+                                data-valid-mount="$ctrl.job.volumes"
+                                data-ng-model="mountPath"
+                                data-ng-pattern="/^\/(\S)*/"
+                                data-ng-model-options="{ debounce: 500 }"
+                                required
+                            />
+                        </oui-field>
+                    </oui-inline-adder-field>
+                    <oui-inline-adder-field data-adaptive>
+                        <oui-field
+                            data-label="{{:: 'pci_projects_project_training_jobs_submit_data_permission' | translate  }}"
+                            size="m"
+                        >
+                            <label class="oui-select">
+                                <select
```

Comment:
`oui-select` can't be used as a component ?

Reply:
https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-select-native--inline#inline
I used this one. Not ok ?
I need to translate all field

Reply:
I wasn't aware that ui-kit documentation provide examples with only CSS classes, my bad.
However by using a `oui-field`, aren't there two labels for the select in that case ?